examples/expressions/10_expressions_intro_new.py

```diff
@@ -0,0 +1,201 @@
+"""
+.. _example_expressions_intro:
+
+
+Building powerful machine learning pipelines with the skrub DataOps
+=========================
```

Comment:
```suggestion
Introduction to machine learning pipelines with skrub DataOps
==============================================================
```

Reply:
I'm changing the title to "introduction", rather than "powerful", because the pipeline here is quite simple.

We need to point to an example with a more complex pipeline at the end, saying that in such settings DataOps unlock things that are impossible with scikit-learn pipelines.